presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -169,14 +168,18 @@ public void testComplexCommonFilterPushdown()
                 "  LEFT JOIN high on low.datestr = high.datestr" +
                 " ORDER BY\n" +
                 "    ofin.order_key\n";
-        compareResults(queryRunner.execute(Session.builder(super.getSession())
-                                .setSystemProperty(PUSHDOWN_SUBFIELDS_ENABLED, "true")
-                                .setSystemProperty(CTE_MATERIALIZATION_STRATEGY, "HEURISTIC_COMPLEX_QUERIES_ONLY")
-                                .setSystemProperty(CTE_FILTER_AND_PROJECTION_PUSHDOWN_ENABLED, "true")
-                                .build(),
-                        testQuery),
-                queryRunner.execute(getSession(),
-                        testQuery));
+        Session materializedSession = Session.builder(super.getSession())
+                .setSystemProperty(VERBOSE_OPTIMIZER_INFO_ENABLED, "true")
+                .setSystemProperty(PUSHDOWN_SUBFIELDS_ENABLED, "true")
+                .setSystemProperty(CTE_MATERIALIZATION_STRATEGY, "HEURISTIC_COMPLEX_QUERIES_ONLY")
+                .setSystemProperty(CTE_FILTER_AND_PROJECTION_PUSHDOWN_ENABLED, "true")
+                .build();
+        verifyResults(materializedSession, getSession(), queryRunner, testQuery,
```

Comment:
@jaystarshot please note that only the is_materialized value for the cte named "order_platform_data" is false, while the other 4 cte ("urgent", "medium", "high", "low") are all false.